changelog.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## v1.0.1 2020-02-25
```

Comment:
FYI new features should be minor versions, not patches

Reply:
Sorry I didn't see guidance anywhere, feel free to correct it